test.py

```diff
@@ -0,0 +1,98 @@
+from boltiot.bolt import Bolt
+import unittest
+import json
+import time
+
+client = Bolt('<your-api-key-here>','<your-bolt-id>') # Pass in the API Key and the client ID.
+
+class BoltTests(unittest.TestCase):
+    
+# Testing the digital write function.
+    
+    def test_1(self):
+        assert_value = json.loads(client.digitalWrite('4','HIGH'))
+        self.assertEqual(assert_value["success"],"1")
+        self.assertEqual(assert_value["value"],"1")
+        print("Digital Write Successfull!")
+
+# Testing the analog write function
+        
```

Comment:
@vimalsheoran Why is test-3 is placed before test-2

Reply:
@Deepanshu625  It has been fixed. Thanks for pointing out.